src/signature.rs

```diff
@@ -200,17 +200,13 @@ ecdsa!(ECDSA_P384_SHA512, "P-384 (secp384r1)", ecc::EC_GROUP_P384, "SHA-512",
 
 
 /// EdDSA signatures.
-struct EdDSA {
-    _unused: u8, // XXX: Stable Rust doesn't allow empty structs.
-}
+struct EdDSA;
```

Comment:
Alternatively, this could be written:

```rust
struct EdDSA { }
```

...which was not stable in 1.0 (https://github.com/rust-lang/rfcs/pull/218).